src/H5VLprivate.h

```diff
@@ -27,28 +27,33 @@
 /* Library Private Macros */
 /**************************/
 
+/* If the module using this macro is allowed access to the private variables, access them directly */
+#ifdef H5VL_MODULE
+#define H5VL_OBJ_RC(VOL_OBJ)         ((VOL_OBJ)->rc)
```

Comment:
Getters, for the rest of the library packages.  (Similar idiom as in H5Fprivate.h)